app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ActiveRecord::Base
   has_secure_password validations: false
   has_many :reviews
-  has_many :queue_items, -> { order("created_at ASC") }
+  has_many :queue_items, -> { order('list_order ASC') }
```

Comment:
@tomtomecek Yea, for some reason I was getting a failed test here without the ASC.  I then realized if I changed it from 'list_order' to :list_order , then it would pass.